Feat(content): add Unfettered raids in Unfettered space, Wah Ki and bordering systems (#8784)
endless-sky/endless-sky: data/governments.txt

	"hostile hail" "hostile hai"
	"hostile disabled hail" "hostile disabled hai"

> # A government used to have Unfettered raids in Wah Ki and nearby systems.

Reviewer: ```suggestion
# A government that is used to allow Unfettered raids in Wah Ki and nearby systems.
```

Not really incorrect, but "used to have" also implies previously but not longer possessing, so more clarity might prevent future confusion. :)

government "Hai (Raided)"
	swizzle 0
	color "governments: Hai"
	
	"player reputation" 0
	"attitude toward"
		"Hai (Wormhole Access)" 1
		"Hai Merchant" 1
		"Hai Merchant (Sympathizers)" 1
		"Hai Merchant (Human)" 1
		"Hai (Unfettered)" -.1
		"Hai (Friendly Unfettered)" .1
		"Hai (Unfettered Civilians)" .1
		"Merchant" .01
		"Elenctic Commune" .1

> 	"bribe" .2

Reviewer: ```suggestion
	"bribe" .02
```

Just noticed this, too.

To match the others. Or is this deliberately different and not a typo?

Author: look above, the main hai gov has .2

	"friendly hail" "friendly hai"
	"friendly disabled hail" "friendly disabled hai"
	"hostile hail" "hostile hai"
	"hostile disabled hail" "hostile disabled hai"
	raid "Large Plundering Unfettered" 10

> 

Author: If we use the sympathizers we'd make farming UHai harder and probably make ppl realise smth is going on?

> 

Author: I'd go with 2. but it'd be nice to have other opinions
It would convey Hai dont like you killing UHai over their worlds that much?

	"friendly disabled hail" "friendly disabled hai"
	"hostile hail" "hostile hai"
	"hostile disabled hail" "hostile disabled hai"

> 	# 3 Geocoris woth of undefended cargo

Author: ```suggestion
	# 3 Geocoris worth of undefended cargo
```

> 	#TODO: make it 4x worse and only SBs and sea scorps, full of ion/tracker/pulse

Author: ```suggestion
	# TODO: make it 4x worse and only SBs and sea scorps, full of ion/tracker/pulse
```
